pkg/nodemanager/nodemanager.go

```diff
@@ -61,6 +57,50 @@ type NodeProvider interface {
 	GetZone(ctx context.Context) (cloudprovider.Zone, error)
 }
 
+// labelReconcileInfo lists Node labels to reconcile, and how to reconcile them.
+// primaryKey and secondaryKey are keys of labels to reconcile.
+//   - If both keys exist, but their values don't match. Use the value from the
+//   primaryKey as the source of truth to reconcile.
+//   - If ensureSecondaryExists is true, and the secondaryKey does not
+//   exist, secondaryKey will be added with the value of the primaryKey.
+var labelReconcileInfo = []struct {
+	primaryKey            string
```

Comment:
do you have the design doc about this label reconciler logic?

Reply:
this is same logic from k/k cloud-node-controller. Since some node lables goes to GA, we should do the same for it.

Refer https://github.com/kubernetes/kubernetes/pull/81431.